astroplan/tests/test_scheduling.py

```diff
@@ -21,70 +22,182 @@
                                      dec=89.26410897 * u.deg), name="Polaris")
 
 apo = Observer.at_site('apo')
-targets = [vega, rigel, polaris]
+targets = [vega, polaris, rigel]
+default_time = Time('2016-02-06 03:00:00')
```

Comment:
That default time is used in all of the tests, not just the insert_slot, would it be better to create the Time object separately for each test?